opm/material/densead/DynamicEvaluation.hpp

```diff
@@ -87,8 +88,8 @@ class Evaluation<ValueT, DynamicSize>
     void checkDefined_() const
     {
 #ifndef NDEBUG
-       for (const auto& v: data_)
-           Valgrind::CheckDefined(v);
+        for (int i = dstart_(); i < dend_(); ++i)
```

Comment:
I have not made it work for range for loop yet. that is why I changed the code here a little bit.

Reply:
that's okay. IMO ranged loops are overrated anyway ;)

Reply:
Sure. I do not add this unless requested later.  It is a class should only used by DynamicEvaluation.